client/src/interface/Profile.ts

```diff
@@ -13,9 +13,20 @@ export interface Profile {
   address: string;
   phone: string;
   description: string;
-  birth: Date;
+  birth: string;
   title: string;
   hourlyWage: number;
   createdAt: Date;
   updatedAt: Date;
 }
+
+export interface ICreateProfile {
```

Comment:
```suggestion
export interface CreateProfile {
```
There is a debate whether or not to prefix interfaces with 'I' and it is recommended not to do so in JS 😄